src/main/scala/vectorpipe/osm/Element.scala

```diff
@@ -17,7 +17,7 @@ sealed trait Element {
 private[vectorpipe] object Element {
   implicit val tag: Parser[(String, String)] = (
     Parser.forMandatoryAttribute("k") ~
-      Parser.forMandatoryAttribute("v")
+    Parser.forMandatoryAttribute("v")
```

Comment:
`v` may be `""` (and `k`, too, under rare circumstances).

Reply:
What would the `<tag ...>` look like in that case?

Reply:
```xml
<tag k="" v="" />
```

Reply:
So, depends on what "mandatory" means...

Reply:
Oh, in that case we're all good. `mandatory` here just means that the `k` and `v` *attributes* must exist or the parse should fail.